cfme/tests/configure/test_access_control.py

```diff
@@ -3201,3 +2981,45 @@ def test_chargeback_report():
 
     """
     pass
+
+
+@pytest.mark.customer_scenario
+@pytest.mark.tier(2)
+@pytest.mark.meta(coverage=[1836125])
+def test_ansible_playbook_stdout():
+    """ Test standard output of ansible playbook of particular user
+    Bugzilla:
+        1836125
+    Polarion:
+        assignee: dgaikwad
+        initialEstimate: 1/4h
+        caseimportance: high
```

Comment:
@dgaikwad Add corresponding `casecomponent` and `tags`

Reply:
@Gauravtalreja1 thanks for the suggestion, changes are done.